### PR TITLE
Added import constraints linked to other variables. E.g. coking coal coal import in India.

### PR DESCRIPTION
**Pull request by nicobauer**
## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [ ] Bug fix 
- [ ] Refactoring
- [ x] New feature 
- [x ] Minor change (default scenarios show only small differences)
- [ ] Fundamental change
- [ ] This change requires a documentation update

## Checklist:

- [x ] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x ] I performed a self-review of my own code
- [x ] I explained my changes within the PR, particularly in hard-to-understand areas
- [ ] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [ ] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [ ] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [ x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [ ] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):

* Test runs are here: /p/projects/remind/users/nicolasb/remind3p2p2_prep/remind-1/output/SSP2-Base_2024-08-30_10.20.07
* Comparison of results (what changes by this PR?):

### CODE REVIEW

File: modules/24_trade/standard/input/p24_trade_constraints.cs4r [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*NB* by hand, no mrremind or similar
```
Comment (LaviniaBaumstark) [inline modules/24_trade/standard/input/p24_trade_constraints.cs4r:1]: this file needs to be generated by mrremind. Can be part of the source expert_guess if there is no other source for this information

**Review by LaviniaBaumstark — CHANGES_REQUESTED** [review]

**Comment by nicobauer** [comment]
Never did that. No idea. #FormFollowsFunction

**Comment by nicobauer** [comment]
@LaviniaBaumstark Du sag mal eigentlich müsste dieser PR doch nun akzeptiert sein. Oder?

**Comment by LaviniaBaumstark** [comment]
still waiting for the new input data :-)

**Comment by nicobauer** [comment]
@LaviniaBaumstark I can now merge all the changes into  develop. Right? I think you still need to approve the changes in the review process.

**Comment by LaviniaBaumstark** [comment]
you first need to commit your adjustments to the default.cfg

**Comment by nicobauer** [comment]
Es ist mir ein Buch mit 7 Siegeln. Ich habe jetzt einen extra PR für die revsions-Nummer gemacht. [Welcher Sadist hat sich nur dieses git-System ausgedacht?]

**Comment by LaviniaBaumstark** [comment]
it should work as part of this PR, but o.k. merging just back to back is also fine for now.

**Review by LaviniaBaumstark — APPROVED** [review]

**Pull request merged**